.Rhistory

```diff
@@ -0,0 +1,3 @@
+db_playerids <- readRDS("~/Fantasy Football/dynastyprocess_data/files/db_playerids.rds")
```

Comment:
can you remove this .Rhistory file? should be `git rm .Rhistory`

Reply:
Done. I didn't realize that was in there, thanks.